releasenotes/README.md

```diff
@@ -26,12 +26,38 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.21.7 (Dec 13, 2023) [stable-rc]
+## 2.22.7 (Dec 20, 2023) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
 ------------
 
+* NEW: These scratch org snapshot commands are now beta; they were previously pilot. 
```

Comment:
I usually say "in beta" - but looks like you don't. So if that's your convention, carry on.

Reply:
I never know how to say it.  "in beta" sounds okay, but "in pilot" doesn't. to me.  but I might change it anyway. Thx!